File: PamelloV6.Web/src/services/api/model/search/YoutubeSearchResult.ts
import { IPamelloSong } from "../PamelloSong";
import { IYoutubeSearchVideoInfo } from "./YoutubeSearchVideoInfo";

export interface IYoutubeSearchResult {
    resultsCount: number;
    pamelloSongs: IPamelloSong[];
    youtubeVideos: IYoutubeSearchVideoInfo[];
	query: string;
}

File: PamelloV6.API/Model/Youtube/YoutubeSearchResult.cs
namespace PamelloV6.API.Model.Youtube
{
    public class YoutubeSearchResult
    {
        public int ResultsCount;
        public List<PamelloSong> PamelloSongs;
        public List<YoutubeSearchVideoInfo> YoutubeVideos;

        public YoutubeSearchResult() {
            ResultsCount = 0;
            PamelloSongs = new List<PamelloSong>();
            YoutubeVideos = new List<YoutubeSearchVideoInfo>();
        }
    }
}
